mne/simulation/source.py

```diff
@@ -218,3 +220,126 @@ def simulate_stc(src, labels, stc_data, tmin, tstep, value_fun=None):
 
     stc = SourceEstimate(data, vertices=vertno, tmin=tmin, tstep=tstep)
     return stc
+
+
+def simulate_stc_from_dipole(dipole, bem, info, trans=None, min_dist=5.0,
```

Comment:
any idea of a better name than simulate_stc_from_dipole ?

make_forward_dipole ?

than an

apply_forward ?

just thinking out loud...